File: web/src/views/private/Centro/validations/centro.ts
import * as Yup from "yup";

export const centroSchema = Yup.object({
  nombre: Yup.string().required("Nombre es requerido"),
  idMunicipio: Yup.number().required("El municipio es requerida"),
  direccion: Yup.string().required("Direccion es requerido"),
  area: Yup.string().required("Area es requerido"),
  seguimientoCasos: Yup.string(),
  contacto: Yup.string().required("contacto es requerido"),
  telefono: Yup.string(),
});

export interface CentroForm extends Yup.InferType<typeof centroSchema> {}

File: server/Dtos/Centro.cs
namespace server.Dtos
{
    public class CentroDTO
    {
        public int IdMunicipio { get; set; }
        public required string Direccion { get; set; }
        public required string Nombre { get; set; }
        public required string Area { get; set; }
        public required string? SeguimientoCasos { get; set; }
        public required string Contacto { get; set; }
        public required string? Telefono { get; set; }

    }
}